src/textual/css/_help_text.py

```diff
@@ -0,0 +1,690 @@
+from __future__ import annotations
+
+import sys
+from dataclasses import dataclass
+
+from textual.css._help_renderables import Example, Bullet, HelpText
+from textual.css.constants import (
+    VALID_BORDER,
+    VALID_LAYOUT,
+    VALID_EDGE,
+    VALID_ALIGN_HORIZONTAL,
+    VALID_ALIGN_VERTICAL,
+    VALID_STYLE_FLAGS,
+)
+
+if sys.version_info >= (3, 8):
+    from typing import Literal, Iterable
+else:
+    from typing_extensions import Literal
+
+if sys.version_info >= (3, 9):
+    from typing import Annotated
+else:
+    from typing_extensions import Annotated
+
+from textual.css._error_tools import friendly_list
+from textual.css.scalar import SYMBOL_UNIT
+
+StylingContext = Annotated[
```

Comment:
What's the `Annotated` doing? I know it attaches meta data to types, but what uses the string? Does Mypy show it on error?

Reply:
I didn't know how to docstring a module level variable and Google yielded absurd amounts of conflicting info. I've changed to """ underneath the variable now. I don't know how or where mypy uses this.

Reply:
It's unfortunate there is no docstring equivalent for such things.